### PR TITLE
Ft desktop version

### PR DESCRIPTION
**Pull request by freddykaberuka**
## Completed Task

- implement Media Queries on Mobile version to Desktop version

## ScreenShot
screen1
screen2
screen3
screen4

### CODE REVIEW

File: app.css [hunk 1]
```diff
@@ -267,15 +272,133 @@ footer {
...
+@media screen and (max-width: 720px) {
+  .nav-link {
+    display: none;
+  }
+  .logo-info {
+    display: block;
+  }
+}
+
+@media screen and (min-width: 720px) {
```
Comment (TedLivist) [inline app.css]: - [x] For the desktop version the `min-width` should be from 768px upwards. Kindly fix that.

File: app.css [hunk 2]
```diff
@@ -267,15 +272,133 @@ footer {
...
+    text-align: center;
+  }
+  .contact-form textarea {
+    justify-content: center;
+    padding: 10px 10px;
+    width: 400px;
+  }
+  .contact-button {
+    justify-content: center;
+  }
```
Comment (TedLivist) [inline app.css]: There appears to be so much disparity between the Figma template and your implementation. Take a look at the images below. From about 720px screen width up to a certain width, a lot of your elements have issues.

- [x] You entire website has some space to the right that is unnecessary as can be seen from the Intro section to the Contact Me section
- [x] The Intro section of your design has a lot of spaces on the right. Kindly fix that.
- [x] The Work cards:
    - Have a lot of spaces below the images.
    - The text on the work cards should have some spaces below them. There should be equal margin on both sides of the cards.

  **Kindly fix these issues for ALL your cards.**
- [x] In the About me sections, the white background should cover all the contents. Kindly fix that.
- [x] Kindly ensure that the paragraph in the Contact section occupies some more width to look like what we have on the Figma template.
- [x] Ensure that the background of the Contact Me section covers all its contents.

  First Image | Second Image
  :---:|:---:
  freddy-1 | freddy-2

File: index.html [hunk 3]
```diff
@@ -50,8 +55,14 @@ <h3>Let's CONNECT</h3>
         </section>
         <section class="portofolio">
             <div class="work-container">
```
Comment (TedLivist) [inline index.html:58]: - [x] For your entire Work Section kindly ensure that your images are alternating as we have on the template. For example, in the Figma template, the first image is on the left, the second on the right, the third on the left, the fourth on the right. Kindly implement that.

  Your Implementation | Figma Design
  :---:|:---:
  freddy-str | freddy-alt

File: app.css [hunk 4]
```diff
@@ -267,15 +272,133 @@ footer {
...
+  .logo-info {
+    display: block;
+  }
+}
+
+@media screen and (min-width: 720px) {
+  .personal-info-detail p {
+    margin: 10px 100px;
+  }
```
Comment (TedLivist) [inline app.css]: - [x] In the Intro section of your portfolio, kindly ensure you have the appropriate background image as we have on the template.

- [x] Kindly make your **`'I'm Freddy Glad to see to you'`** text bigger, as we have on the template

- [x] The **'Glad to see you!'** text should be on the next line as we have on the template, kindly do that.

- [x] As I have pointed out earlier, kindly ensure you remove the unnecessary space on the right side of you website.
  Your Implementation | Figma Design
  :---:|:---:
  freddy-intro | freddy-intro-figma

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @freddykaberuka,

Good job so far!
✔️ Good documentation
✔️ Use of flexbox and grid
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (sja-thedude) [reply index.html:58]: Good job in implementing the changes.

File: app.css (hunk 2 above)
Comment (sja-thedude) [reply app.css]: Good job in implementing the changes.

File: app.css (hunk 1 above)
Comment (sja-thedude) [reply app.css]: Good job in implementing the changes.

File: app.css [hunk 5]
```diff
@@ -51,15 +54,17 @@ body {
   text-decoration: none;
   color: #172b4d;
   font-style: normal;
-  font-size: 40px;
   /* stylelint-disable-next-line csstree/validator */
   font-weight: 700, bold;
 }
 .personal-info-detail p {
```
Comment (sja-thedude) [inline app.css:66]: - [x] As suggested by the previous reviewer, Kindly ensure you have the appropriate background image as we have on the template in the Intro section of your portfolio.

 - [x] As suggested by the previous reviewer, kindly place the 'Glad to see you!' text on the next line as we have on the template, kindly do that.

- Your design:
Screenshot 2022-04-02 at 6 45 37 AM

- Figma design:
Screenshot 2022-04-02 at 6 45 03 AM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @freddykaberuka, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Great on implementing the changes as asked.

 ## Required
- [ ] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -48,10 +53,49 @@ <h3>Let's CONNECT</h3>
...
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                </div>
+                <div class="work-button"><button>See Project</button></div>
+                </div>
+                </div>
+            </div>
+            <div class="desktop-work">
```
Comment (prolajumokeoni) [inline index.html:90]: ### OPTIONAL

- [ ] Kindly adjust the images and description for each  card in your desktop-work section to look like the Figma design, Each card is divided half of it contains the project's picture and the other half contains the project's description


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
Screenshot_39  | Screenshot_38

File: app.css [hunk 7]
```diff
@@ -267,15 +283,129 @@ footer {
...
   background-color: #4c73d6;
   color: white;
 }
-.contact-button:disabled {
+.cont-button:disabled {
   background-color: #6276a8;
   color: white;
 }
+
+@media screen and (max-width: 768px) {
```
Comment (prolajumokeoni) [inline app.css:299]: ### OPTIONAL
- [ ] Kindly use min-width instead of max width, this helps you target screen sizes larger the 768px 🤝

**Review by prolajumokeoni — APPROVED** [review]
Hi @freddykaberuka, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_Check inline comments_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**